apps/docs/content/components/Box/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Box
+description: Use a box to simplify the process of applying one-off styles to an element
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![Three empty cards inside a grey background with rounded corners, which is also offset from the edge and including several annotations. First indicating distance between the background area and canvas edge to indicate margin controls. Second showing padding controls between the cards and background outer edge. Third pointing towards background color and lastly one pointing at the rounded corner to illustrate border radius options.](https://github.com/primer/brand/assets/13340707/211be3c8-0f33-47ba-8236-9b19780d2a86)
```

Comment:
We normally use pointers for parts or sub elements in the component, but not for its options. I think that for this specific component we might want to skip the intro image, as it seems a bit difficult to express such an abstract component/wrapper.


```suggestion
![Three empty cards inside a grey background with rounded corners, which is also offset from the edge and including several annotations. First indicating distance between the background area and canvas edge to indicate margin controls. Second showing padding controls between the cards and background outer edge. Third pointing towards background color and lastly one pointing at the rounded corner to illustrate border radius options.](https://github.com/primer/brand/assets/13340707/211be3c8-0f33-47ba-8236-9b19780d2a86)
```

Reply:
👍 I can remove it. Feels quite text heavy though, any other way we can help folks quickly visualise the usefulness of this component?

Reply:
I could think of adding real use case examples, as I did for [Stack](https://primer.style/brand/components/Stack/). Maybe on a follow up PR? just to unblock this PR release